helm/checksum_annotations.md

```diff
@@ -0,0 +1,43 @@
+# Helm Checksum Annotations
+
+## Overview
+
+- We add checksum annotations to our deployments and daemonsets to trigger
+fresh pods when their configmaps or secrets have changed.
+- This reduces how often we need to roll pods which can be disruptive.
+- Changes to other resources like PSP and RBAC are applied without needing to
+roll pods.
+- If the app has auto reload functionality like Nginx Ingress Controller these
+annotations are not needed.
+
+## Annotations
+
+- The secret annotation is only needed if Chart has a template.
```

Comment:
I don't understand this sentence - can you rephrase it? Does it mean if the chart has a templated secret?

Reply:
Yes that's what I meant :) but this is much better.